rest/src/main/scala/io/udash/rest/companions.scala

```diff
@@ -150,4 +150,7 @@ abstract class RestServerOpenApiImplCompanion[Implicits, Real](protected val imp
 
   final def asHandleRequest(real: Real): RawRest.HandleRequest =
     RawRest.asHandleRequest(real)
+
+  final def asHandleRequestWithStreaming(real: Real): RawRest.HandleRequestWithStreaming =
+    RawRest.asHandleRequestWithStreaming(real)
```

Comment:
Add to other companions as well for consistency with some ScalaDoc explaining why there are two methods ?